tests/terraform/checks/resource/alicloud/example_OSSBucketPublic/main.tf

```diff
@@ -0,0 +1,14 @@
+resource "alicloud_oss_bucket" "good-bucket" {
+  bucket = "bucket-170309-acl"
+  acl    = "private"
```

Comment:
please also add a test case with no `acl` set, which defaults to `private` and therefore should pass

Reply:
Added